Source/cursor.cpp

```diff
@@ -189,7 +189,7 @@ void CheckCursMove()
 	sy = MouseY;
 
 	if (chrflag || questlog) {
-		if (sx >= SCREEN_WIDTH / 4) {
+		if (sx >= SCREEN_WIDTH / 4) { /// BUGFIX: (sx >= SCREEN_WIDTH / 4)

```

Comment:
what is the bugfix? `sx >= SCREEN_WIDTH / 4` is used both in the if-statement, and in the `BUGFIX` comment. what is the correct line?

Reply:
Oops should have been 2 like the test for panels on the opposite side 🙂

Reply:
Right. Could you explain briefly why they check of `SCREEN_WIDTH / 2` in the if-statement, but then use `SCREEN_WIDTH / 4` in the `sx -= SCREEN_WIDTH / 4;` and `sx += SCREEN_WIDTH / 4;` lines? Just so I understand what the logic is intending to do.

Reply:
The panels take up half the screen width. But the view area still needs to be centered. So only render haft as much, but do so by removing a quarter from each side of the view area.